_ide_helper.php

```
@@ -1832,6 +1832,34 @@ public static function forgetScopedInstances()
+        /**
+         * Set the callback which determines the current container environment.
+         *
+         * @param (callable(array<int, string>|string): bool|string)|null $callback
+         * @return void
+         * @static
+         */
+        public static function resolveEnvironmentUsing($callback)
+        {
+            //Method inherited from \Illuminate\Container\Container 
+            /** @var \Illuminate\Foundation\Application $instance */
+            $instance->resolveEnvironmentUsing($callback);
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$instance' which will lead to PHP notices. (Clean Code Rules)

(UndefinedVariable)